### PR TITLE
Feat/add script to get api endpoints

### LINKED ISSUE #57: Add documentation focusing on security
Clients have questins regarding security practices needed for integration with defactor, this should be included in the developer documentation

### LINKED ISSUE #81: Generate Markdown Content based on the postman JSON file
- [x] Generate Markdown Content based on the postman JSON file 
- [x] Import Markdown Content into Docusaurus 
- [ ] Import docusaurus content into shared GPT assistant
- [ ] use GPT assistant to edit / review content in the docs  
 

Document API in **https://defactor.dev/docs/category/back-end-services**

### PR DESCRIPTION
**Pull request by Torresmorah**
### What does this PR do?

- Resolve #81
- Resolve #57
- Add security and role-based authentication details

### Steps to test

1. Check the content

### Screenshots

imagen
imagen
imagen

### CODE REVIEW

File: docs/back-end-api/erc20CollateralToken/graphql.md [hunk 1]
```diff
@@ -96,8 +120,8 @@ Upon successful completion of a request, the server will issue a status code of
 {
   "v1RestoreSession": {
     "res": {
-      "accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWUsImlhdCI6MTUxNjIzOTAyMiwiaHR0cHM6Ly9oYXN1cmEuaW8vand0L2NsYWltcyI6eyJ4LWhhc3VyYS1kZWZhdWx0LXJvbGUiOiJ1c2VyIiwieC1oYXN1cmEtYWxsb3dlZC1yb2xlcyI6WyJ1c2VyIiwiYWRtaW4iXSwieC1oYXN1cmEtdXNlci1pZCI6IjEyMyIsIngtaGFzdXJhLW9yZy1pZCI6IjQ1NiIsIngtaGFzdXJhLWN1c3RvbSI6ImN1c3RvbS12YWx1ZSJ9fQ.07mlUOhH3Oigz_Yyil8EC579Ht6PbZ1yr8fYJfhQ4NE",
-      "refreshToken": "d9438293-d378-4561-8b67-d7a4608b7d7c"
+        "accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWUsImlhdCI6MTUxNjIzOTAyMiwiaHR0cHM6Ly9oYXN1cmEuaW8vand0L2NsYWltcyI6eyJ4LWhhc3VyYS1kZWZhdWx0LXJvbGUiOiJ1c2VyIiwieC1oYXN1cmEtYWxsb3dlZC1yb2xlcyI6WyJ1c2VyIiwiYWRtaW4iXSwieC1oYXN1cmEtdXNlci1pZCI6IjEyMyIsIngtaGFzdXJhLW9yZy1pZCI6IjQ1NiIsIngtaGFzdXJhLWN1c3RvbSI6ImN1c3RvbS12YWx1ZSJ9fQ.07mlUOhH3Oigz_Yyil8EC579Ht6PbZ1yr8fYJfhQ4NE",
```
Comment (xavier506) [inline docs/back-end-api/erc20CollateralToken/graphql.md:123]: this is a fake access token?

File: docs/back-end-api/erc20CollateralToken/restful.md [hunk 2]
```diff
@@ -0,0 +1,1259 @@
...
+
+**Response**
+
+Upon successful completion of a request, the server will issue a status code of 200 along with a JSON object encapsulating the generated access token and refresh token. This object encompasses the following attributes:
+
+```json
+{
+  "v1Login": {
+    "res": {
+      "accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWUsImlhdCI6MTUxNjIzOTAyMiwiaHR0cHM6Ly9oYXN1cmEuaW8vand0L2NsYWltcyI6eyJ4LWhhc3VyYS1kZWZhdWx0LXJvbGUiOiJ1c2VyIiwieC1oYXN1cmEtYWxsb3dlZC1yb2xlcyI6WyJ1c2VyIiwiYWRtaW4iXSwieC1oYXN1cmEtdXNlci1pZCI6IjEyMyIsIngtaGFzdXJhLW9yZy1pZCI6IjQ1NiIsIngtaGFzdXJhLWN1c3RvbSI6ImN1c3RvbS12YWx1ZSJ9fQ.07mlUOhH3Oigz_Yyil8EC579Ht6PbZ1yr8fYJfhQ4NE",
```
Comment (xavier506) [inline docs/back-end-api/erc20CollateralToken/restful.md:61]: is this a fake access token?

File: docs/back-end-api/erc20CollateralToken/graphql.md (hunk 1 above)
Comment (Torresmorah) [reply docs/back-end-api/erc20CollateralToken/graphql.md:123]: yes, it is

File: docs/back-end-api/erc20CollateralToken/restful.md (hunk 2 above)
Comment (Torresmorah) [reply docs/back-end-api/erc20CollateralToken/restful.md:61]: yes, it is

**Pull request merged**